### PR TITLE
19 sponsors

### LINKED ISSUE #19: Assign an organization to an event as a sponsor
As an organizer, I'd like to be able to select companies as sponsors for an evenment

    Company name
    Logo
    Description of the sponsorship

### PR DESCRIPTION
**Pull request by steakunderscore**
Should close #19. Although as @sophiedeziel mentioned to me, it could be nice to open a new ticket for having the ability to add a sponsors message.

### CODE REVIEW

**Comment by steakunderscore** [comment]
Will fix the tests tomorrow.

File: Gemfile [hunk 1]
```diff
@@ -71,6 +71,7 @@ group :development, :test do
   gem "guard-rspec", require: false
   gem "guard-rubocop"
   gem "pry"
+  gem "dotenv-rails"
```
Comment (cefigueiredo) [inline Gemfile]: Why install `dotenv` if there are not a `.env` file to load?

Comment (cefigueiredo) [inline Gemfile]: I mean, there are no `.env.sample` file nor instruction that the environment variables should be inside a `.env` file...

File: README.md [hunk 2]
```diff
@@ -109,13 +114,16 @@ file, replacing the inside of the quotes with your key/secret:
 
 ```
 export GITHUB_KEY="your_key"
-export GITHUB_SECRET="your_secret"
+export GITHUB_GITHUB_SECRET="your_secret"
+export CLOUDINARY_URL="cloudinary://your_key:your_secret"
```
Comment (cefigueiredo) [inline README.md]: On adopting `.env` strategy, you could generate a `.env.sample` file with the variables, and replace the instructions 

```
 On Unix-based systems using the bash shell, add the following to your `~/.bashrc` 
 file, replacing the inside of the quotes with your key/secret:
```
for something like:
```
Copy `.env.sample` file to `.env` and fill appropriately the variables settings. 
```

File: app/models/event_sponsor.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class EventSponsor < ActiveRecord::Base
```
Comment (sophiedeziel) [inline app/models/event_sponsor.rb]: I'd call that `Sponsorship` because it's a resource.

Comment (steakunderscore) [inline app/models/event_sponsor.rb]: Makes sense. Will update.

**Comment by sophiedeziel** [comment]
Just to let you know that @kiramclean's pull request #210 introduces some changes on the views that will break yours. I highly recommend looking at them. You should be part of the review process on that.

**Comment by sophiedeziel** [comment]
@steakunderscore did you receive my message on slack last week with the test file?

**Comment by steakunderscore** [comment]
Shifted this branch to the Montreal.rb repo. So have to close this PR and open a [new one](https://github.com/montrealrb/Montreal.rb/pull/220)

**Pull request closed without merging**